caikit/runtime/http_server.py

```diff
@@ -221,6 +221,30 @@ def _bind_routes(self, service: ServicePackage):
             else:
                 self._add_unary_input_unary_output_handler(rpc)
 
+    def _get_request_params(
+        self, rpc: CaikitRPCBase, request: "pydantic_request"
+    ) -> Dict[str, Any]:
+        """get the request params based on the RPC's req params"""
+        request_kwargs = dict(request)
+        required_params = rpc.task.get_required_parameters(False)
```

Comment:
nit: the rpc itself should hold the info about whether the input is streaming or not, and maybe that should be used here instead of hardcoding false?

Might at least be worth a comment saying
```
# Here we assume the input isn't streaming
```

Reply:
Good point!